doc-content/enterprise-only/project-data/migrating-legacy-designer-proc.adoc

```diff
@@ -0,0 +1,41 @@
+[id='migrating-from-legacy-designer-proc']
+
+= Migrating business processes to the new designer
+
+The legacy process designer in {CENTRAL} is deprecated in {PRODUCT} {PRODUCT_VERSION_LONG}. It will be removed in a future {PRODUCT} release. The legacy process designer will not receive any new enhancements or features. Existing users that intend to utilize the functionality in the new designer should start migrating their processes to the new designer. All new processes should be created using the new process designer.
+
+[NOTE]
+====
+The {PROCESS_ENGINE} will continue to support the execution and deployment of business processes generated with the legacy designer in to the {KIE_SERVER}. If you have a legacy business process that is functioning and that you do not intend to change, it is not mandatory to migrate to the new designer at this time.
+====
+
+Only business processes that contain nodes types that are supported in the new designer can be migrated. More node types will be supported in future versions of {PRODUCT}.
```

Comment:
node types?
passive. Consider
You can migrate only business processes...

Reply:
Fixed :)